editor/test/integration/script_properties_test.clj

```diff
@@ -36,17 +36,6 @@
   (:import [com.dynamo.gameobject.proto GameObject$CollectionDesc GameObject$PrototypeDesc]
            [com.dynamo.lua.proto Lua$LuaModule]))
 
-(defn- unpack-property-declarations [property-declarations]
-  (assert (map? property-declarations))
-  (into {}
-        (mapcat (fn [[entries-key values-key]]
-                  (let [entries (get property-declarations entries-key)
-                        values (get property-declarations values-key)]
-                    (map (fn [{:keys [key index]}]
-                           [key (values index)])
-                         entries))))
-        (vals properties/type->entry-keys)))
-
```

Comment:
I've moved a bunch of helper functions to the `test-util` module so I can use them across all the tests.